resources/misc.cuh

```diff
@@ -35,6 +35,7 @@ struct VCallDataRecord {
     uint32_t offset;
     uint32_t size;
     const void *src;
```

Comment:
Can we rename `src` into `value` given that it takes on a new identity as literal storage location?